modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -290,7 +290,7 @@ In 4.2, signature changed to `db.listLocks() :: (mode :: STRING, resourceType ::
 | xref:reference/procedures.adoc#procedure_dbms_cluster_uncordonServer[`dbms.cluster.uncordonServer()`]
 | label:no[]
 | label:yes[]
-| label:admin-only[]
+| label:admin-only[] label:deprecated[Deprecated in 5.8] use `ENABLE SERVER` instead
```

Comment:
```suggestion
| label:admin-only[] label:deprecated[Deprecated in 5.23] Replaced by xref:clustering/server-syntax.adoc#server-management-syntax[`ENABLE SERVER`].
```

Reply:
The PR in the codebase will be merged in 5.23. Why is the version here 5.8?

Reply:
This procedure should be moved to the section on deprecated procedures.

Reply:
as well as `dbms.cluster.routing.getRoutingTable`

Reply:
"Replaced by" is not really correct. `ENABLE SERVER` was not introduced to replace this, but it was extended that it can this as well. Back in 5.8 - we just forgot to deprecate it. I guess we now need to say 5.23.

Reply:
Anyway - what labels does this PR need?

Reply:
Anyway - what labels does this PR need?

Reply:
I think 5.23.